pkg/util/disk/iscsi.go

```diff
@@ -97,6 +98,13 @@ type Interface interface {
 	Rescan(devicePath string) error
 
 	GetBlockSizeBytes(devicePath string) (int64, error)
+
+	// Both iSCSIMounter and pvMounter call Mount through a SafeFormatAndMount struct
+	// which in turn checks formatting before mounting. MountWithoutFormat calls the mount.Interface.Mount() method
+	// directly.
+	MountWithoutFormat(source string, target string, fstype string, options []string) error
+
+	UnmountDeviceBindAndDelete(path string) error
```

Comment:
The UnmountPath method tries checking the mount target using the isLikelyNotMountPoint() helper function, which cannot detect file-bind mounts. So had to define a new method instead.